contracts/liquidityMining/KyberRewardLocker.sol

```diff
@@ -250,7 +301,12 @@ contract KyberRewardLocker is IKyberRewardLocker, PermissionAdmin {
       totalVesting
     );
 
-    token.safeTransfer(msg.sender, totalVesting);
+    if (token == IERC20Ext(0)) {
```

Comment:
Another way to do this is keep weth in RewardLocker and FairLaunch.
At '_completeVesting' function, check if this token is weth, if yes then unwarp it, and transfer to user. (So we only need minimum changes)

Reply:
yeah thought about it as well, fairlaunch only handles erc20, but don't think it is too much changes to support native token here